pf/internal/check/checks.go

```diff
@@ -78,26 +75,79 @@ func checkIDProperties(sink diag.Sink, info tfbridge.ProviderInfo, isPFResource
 	return nil
 }
 
-func resourceHasRegularID(resource shim.Resource, resourceInfo *tfbridge.ResourceInfo) (bool, string) {
+type resourceError struct {
```

Comment:
These don't look like docs changes - is this intentional?

Reply:
It's not a docs change and it is intentional. I can move the change to a separate PR.